packages/code-du-travail-frontend/src/layout/Header/SkipNavLink.tsx

```diff
@@ -2,13 +2,13 @@ import { FlatList, theme } from "@socialgouv/cdtn-ui";
 import React from "react";
 import styled from "styled-components";
 
-export function SkipNavLink() {
+const SkipNavLink = (): JSX.Element => {
   return (
     <SkipNav as="div">
-      <nav aria-label="liens d'évitements" role="navigation">
+      <nav aria-label="liens d'évitements">
```

Comment:
Pourquoi avoir retirer le role ici ? C'est bien de la navigation 🤔

Reply:
ça trigger un warning sur toutes nos pages parceque l'info est redondante
![Screenshot 2021-10-18 at 10 58 41](https://user-images.githubusercontent.com/4971715/137700766-4b91a6c2-98e6-4077-a2cd-48491d2f574a.png)
 :

Reply:
En fait c'est redondant avec le fait que ce soit sur une balise nav :
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
"Implicit ARIA role | navigation"

ici ils disent bien :
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role
"It is preferable to use the HTML5 <nav> element to define a navigation landmark"